pkg/services/ngalert/api/tooling/Makefile

```diff
@@ -21,21 +23,7 @@ ifneq ($(OS),Windows_NT)
 endif
 
 spec.json spec-stable.json: $(GO_PKG_FILES)
-	# this is slow because this image does not use the cache
-	# https://github.com/go-swagger/go-swagger/blob/v0.27.0/Dockerfile#L5
-	docker run --rm -it \
-		-w /src/$(PATH_DOWN) \
-		-v $$(pwd)/$(PATH_UP):/src \
-		--entrypoint sh \
-		quay.io/goswagger/swagger:$(SWAGGER_TAG) \
-		-c \
-		"swagger generate spec -m -o $@; swagger generate spec -m --include-tag=stable -o spec-stable.json"
-
-ensure_go-swagger_mac:
-	@hash swagger &>/dev/null || (brew tap go-swagger/go-swagger && brew install go-swagger)
-
-spec.json-mac: ensure_go-swagger_mac $(GO_PKG_FILES)
-	swagger generate spec -m -w $(API_DIR) -o spec.json && swagger generate spec -m --include-tag=stable -o ngalert.json
+	SWAGGER_GENERATE_EXTENSION=false $(SWAGGER) generate spec -m -w $(API_DIR) -o spec.json && $(SWAGGER) generate spec -m --include-tag=stable -o spec-stable.json
```

Comment:
`SWAGGER_GENERATE_EXTENSION=false` is for not including the `x-go-name` properties in the output.